helpme/api/views.py

```diff
@@ -43,8 +43,9 @@ def perform_create(self, serializer):
         teams = Team.objects.filter(sites__in=[instance.site])
         instance.teams.set(teams.filter(categories__contains=instance.category))
 
-        if app_settings.MAIL_LIST:
-            self.send_email(serializer, instance)
+        support_email = SupportEmailClass(self.request).get_key_value().get("support_email")
```

Comment:
On thinking further, would it be better to pass in the Site object rather than the request?  So it would be:

`SupportEmailClass(self.request.site)`

or

`SupportEmailClass(get_current_site(self.request))`

Just trying to think if a project does not put the site on the request, how would they use it.

Reply:
you're right- passing in the site is more reusable for other apps. updated!